src/wasm.h

```diff
@@ -479,7 +479,7 @@ typedef struct WasmModuleField {
   union {
     WasmFunc func;
     WasmImport import;
-    WasmExport export;
+    WasmExport wasm_export;
```

Comment:
"export_" instead? I've been using a trailing underscore for names that conflict (see const_, if_, return_ etc. above)